packages/module/src/components/groups/DefaultGroupExpanded.tsx

```diff
@@ -62,6 +68,36 @@ export function computeLabelLocation(points: PointWithSize[]): PointWithSize {
       lowPoints.push(p);
     }
   });
+
+  points.forEach((p) => {
+    const delta = !highPoints ? -Infinity : Math.round(p[1]) - Math.round(highPoints[0][1]);
+    // If the difference is greater than the threshold, update the highest point
+    if (delta < -threshold) {
+      highPoints = [p];
+    } else if (Math.abs(delta) <= threshold) {
+      if (!highPoints) {
+        highPoints = [];
+      }
+      highPoints.push(p);
+    }
+  });
```

Comment:
This code only needs to happen inside the `labelPosition === LabelPosition.top` case.

Reply:
Updated it.